epimodel/exports/epidemics_org.py

```diff
@@ -147,6 +158,14 @@ def extract_smallish_data(
         if un_age_dist is not None:
             d["AgeDist"] = un_age_dist.to_dict()
 
+        if hospital_capacity is not None:
+            capacity = {}
+            for key, value in hospital_capacity.to_dict().items():
+                if not isinstance(value, float) or not np.isnan(value):
+                    capacity[key] = value
+
```

Comment:
Are you trying to drop nans and otherwise having column: value layout? Then I think you can do so with `{col: val.dropna().to_dict() for col, val in df.items()}`